main/FatesInterfaceMod.F90

```diff
@@ -1551,9 +1572,38 @@ subroutine FatesReportParameters(masterproc)
 
       call FatesReportPFTParams(masterproc)
       call FatesReportParams(masterproc)
-      call FatesCheckParams(masterproc)
+      call FatesCheckParams(masterproc,hlm_parteh_model)
       
       return
    end subroutine FatesReportParameters
 
+   ! ====================================================================================
+
+   subroutine InitPARTEHGlobals()
+   
+     ! Initialize the Plant Allocation and Reactive Transport
+     ! global functions and mapping tables
+     
+     select case(hlm_parteh_model)
+     case (1)
+        call InitPRTInstanceAC()
+     case(2)
+        
+        !call InitPRTInstanceACNP()
+        write(fates_log(),*) 'You specified the allometric CNP mode'
+        write(fates_log(),*) 'with relaxed target stoichiometry.'
+        write(fates_log(),*) 'This mode is not available yet.'
```

Comment:
maybe say what mode is currently available, so anyone who gets here by accident knows how to rectify their run?